tmt/steps/provision/testcloud.py

```diff
@@ -339,7 +342,8 @@ class GuestTestcloud(tmt.GuestSsh):
         arch ....... architecture for the VM, host arch is the default
     """
 
-    _data_class = TestcloudGuestData
+    # TODO: revisit this once we get rid of `follow_imports = skip`
+    _data_class = TestcloudGuestData  # type: ignore
```

Comment:
hmm, can you revisit this? seems artemis does not use this ... and that is the role-model

Reply:
Where do you see it? I don't see a `_data_class` property. Mypy is complaining about:
```
tmt/steps/provision/testcloud.py:346: error: Incompatible types in assignment (expression has type "Type[TestcloudGuestData]", base class "GuestSsh" defined the type as "Type[GuestSshData]")  [assignment]
```
I believe enabling `follow_imports` could fix it.

Reply:
Artemis does not set this field at all, and that's a bug (#1435).

`# type: ignore` might be needed to silence mypy, in the future it would be removed with https://github.com/teemtee/tmt/issues/1437

Reply:
Ok then I will keep the `type: ignore` and edit the comment to revisit it once `Guest` becomes a generic type.